bin/export.py

```diff
@@ -165,6 +171,25 @@ def build_problem_zip(problem, output, statement_language):
                     out = problem.name / out
                 copyfiles.add((f, out))
 
+    for pattern, required in testcases:
```

Comment:
There's a bit much duplication here but I suppose we'll live with it. Can you add a comment what this loop does do differently than the one above?